src/coreComponents/common/MpiWrapper.cpp

```diff
@@ -156,45 +156,50 @@ int MpiWrapper::test( MPI_Request * request, int * flag, MPI_Status * status )
 {
 #ifdef GEOSX_USE_MPI
   return MPI_Test( request, flag, status );
-#endif
+#else
```

Comment:
Is it mandatory to get this alongside the Pangea3 option update?

Reply:
I'll separate.